README.rst

```diff
@@ -158,6 +158,14 @@ The process of creating replicas is similar to starting migrations::
     --destination-environment "$DESTINATION_ENV" \
     --instance $VM_NAME
 
+Updating a replica
+------------------
+
+To update a replica::
+
+    coriolis replica update  $REPLICA_ID \
+    --destination-environment "$DESTINATION_ENV"
```

Comment:
Could you please also add the `--source_environment`, `--notes`, `--network_map` and all the storage-related params (namely `--default-storage-backend`, `--disk-storage-mapping` and `--storage-backend-mapping`) for the example?

Reply:
Please ignore this comment as ideally we should make a separate PR to add all the new args.